### DISCUSSION TITLE
modularized :: How to see your masks

### DISCUSSION THREAD

**Discussion by gniknalu** [roflcoopter/viseron#329] [Q&A] [answered]
Is there a way, in the modularized version, to see your masks in the nvr?

Thank you!

**Comment by roflcoopter** [comment, marked as answer]
You can view the MJPEG streams in the browser directly, and choose what is drawn.
Just replace `<host>` and `<camera_identifier>` below

`http://<host>:8888/<camera_identifier>/mjpeg-stream?draw_objects=1&draw_motion=1&draw_motion_mask=1`

**Reply by gniknalu** [reply]
Thank you!!
